src/app/patient-dashboard/common/patient-banner/patient-banner.component.html

```diff
@@ -17,7 +17,10 @@
     <div class="row clearfix">
       <span style="font-weight: bold;font-size:20px">{{ patient?.person.display}}</span>
       <span>{{ patient?.person.gender}}</span>
-      <span>{{ birthdate }} <span *ngIf="birthdate">({{ patient?.person.age }} yo)</span></span>
+      <span>{{ birthdate }} 
+        <span *ngIf="birthdate && patient?.person.age>=18">({{ patient?.person.age }} yo)</span>
+        <span *ngIf="birthdate && patient?.person.age<18">({{ formattedPatientAge }} )</span>
```

Comment:
L21 has a string `yo` shouldn't that be the case for L22?

Reply:
For ages 18 and above they prefer the age to be displayed as 18yo. For ages below 18 they prefer the year, month and days shown, so the formatted string becomes 2y 9m 11d